analytics_data_api/v0/models.py

```diff
@@ -214,6 +216,27 @@ class Meta(BaseVideo.Meta):
 
 
 class RosterEntry(DocType):
+
+    course_id = String()
+    username = String()
+    name = String()
+    email = String()
+    enrollment_mode = String()
+    cohort = String()
+    segments = String(fields={'raw': String()})
+    problems_attempted = Integer()
+    problems_completed = Integer()
+    problem_attempts_per_completed = Float()
+    # Useful for ordering problem_attempts_per_completed (because results can include null, which is
+    # different from zero).  attempt_ratio_order is equal to the number of problem attempts if
+    # problem_attempts_per_completed is > 1 and set to -problem_attempts if
```

Comment:
I'm not sure about this definition of `attempt_ratio_order`. Consider two students:

* Student A has 2 problem attempts, and 1 completion
* Student B has 4 problem attempts and 3 completions

Studen B has a better ratio (and has done more work), but they'd rank further away from the optimal end of the list.

Am I understanding this definition properly? If so, what do you guys think about this case?

@mulby